src/main/webapp/recipe-display.html

```diff
@@ -70,6 +71,18 @@
           })
       }
 
+      function deletePost(recipeID) {
+        // eslint-disable-line no-unused-vars
```

Comment:
Can delete this eslint-disable-line now?

Reply:
Good point